test/langtools/tools/javac/classfiles/attributes/innerclasses/InnerAnnotationsInInnerAnnotationTest.java

```diff
@@ -26,7 +26,6 @@
  * @bug 8042251
  * @summary Testing InnerClasses_attribute of inner annotations in inner annotation.
  * @library /tools/lib /tools/javac/lib ../lib
- * @enablePreview
```

Comment:
I think you should update the copyrights in these files.

Reply:
Good point. [Updated the copyright script](https://github.com/openjdk/jdk/pull/26998) to work for valhalla and done.